api4/emoji.go

```diff
@@ -53,7 +53,7 @@ func createEmoji(c *Context, w http.ResponseWriter, r *http.Request) {
 	defer c.LogAuditRec(auditRec)
 
 	// Allow any user with CREATE_EMOJIS permission at Team level to create emojis at system level
-	memberships, err := c.App.GetTeamMembersForUser(c.AppContext.Session().UserId)
+	memberships, err := c.App.GetTeamMembersForUser(c.AppContext.Session().UserId, true)
```

Comment:
Out of the scope, but this seems weird. Looks like if we used to have permissions to create emojis in a team that we are no longer a member of, we could still create an emoji by hitting the API directly. Not sure if other API endpoints are suffering of similar behaviour, but I feel someone should take a look.

Reply:
Agree. @Willyfrog - should I create a ticket for this?

Reply:
Added: https://mattermost.atlassian.net/browse/MM-44407